rest/data/host.go

```diff
@@ -57,34 +51,6 @@ func (hc *DBHostConnector) FindHostsInRange(apiParams restmodel.APIHostParams, u
 	return hostRes, nil
 }
 
-// FindHostById queries the database for the host with id matching the hostId
-func (hc *DBHostConnector) FindHostById(id string) (*host.Host, error) {
-	h, err := host.FindOne(host.ById(id))
-	if err != nil {
-		return nil, err
-	}
-
-	return h, nil
-}
-
-// FindHostByIP queries the database for the host with ip matching the ip address
-func (hc *DBHostConnector) FindHostByIpAddress(ip string) (*host.Host, error) {
-	h, err := host.FindOne(host.ByIP(ip))
-	if err != nil {
-		return nil, err
-	}
-
-	return h, nil
-}
-
-func (dbc *DBConnector) FindHostByIdWithOwner(hostID string, user gimlet.User) (*host.Host, error) {
-	return findHostByIdWithOwner(dbc, hostID, user)
-}
-
-func (hc *DBHostConnector) FindHostsByDistro(distro string) ([]host.Host, error) {
-	return host.Find(db.Query(host.ByDistroIDsOrAliasesRunning(distro)))
-}
-
 func (hc *DBConnector) GetPaginatedRunningHosts(hostID, distroID, currentTaskID string, statuses []string, startedBy string, sortBy string, sortDir, page, limit int) ([]host.Host, *int, int, error) {
```

Comment:
I don't see that the mock is used here; do we need to keep this as a connector function?

Reply:
Nope, this slipped through the cracks. Thanks!